es/authentication_authorization/README.md

```diff
@@ -143,21 +143,21 @@ Vamos a agregarle algo de dulce a nuestras plantillas mientras estamos ahí. Pri
 
 Esto agrega un lindo "Hello _&lt;username&gt;_" para recordarle al usuario como ingresó, y que está autenticado. También agrega un enlace de salida del blog -- como puedes ver, aún no funciona. ¡Vamos a arreglarlo!
 
-Decidimos apoyarnos en Django para manejar el ingreso, así que vamos a dejar que Django se encargue de la salida. Mira https://docs.djangoproject.com/en/1.10/topics/auth/default/ y ve si encuentras algo.
+Decidimos apoyarnos en Django para manejar el ingreso, así que vamos a dejar que Django se encargue de la salida. Mira https://docs.djangoproject.com/en/2.1/topics/auth/default/ y ve si encuentras algo.
```

Comment:
The Django Girls tutorial (both the English original and the current Spanish translation) [use Django 2.0](https://tutorial.djangogirls.org/en/django_installation/#installing-packages-with-requirements), as does the corresponding link in the English tutorial extensions. Thus I think we should link to the 2.0 documentation here for now, too.

(I would have suggested to link to the Spanish Django documentation at https://docs.djangoproject.com/es/2.0/topics/auth/default/, but its content seems to just be the English one again, untranslated :thinking:)

Reply:
Yeah, I hand't thought about that but you are right, the Spanish version is indeed in English :sweat_smile: